_topic_maps/_topic_map_rosa.yml

```diff
@@ -137,12 +152,10 @@ Topics:
     File: rosa-creating-cluster
 # - Name: Creating a ROSA cluster using the web console
 #   File: rosa-creating-cluster-console
-  - Name: Creating an AWS PrivateLink cluster on ROSA
-    File: rosa-aws-privatelink-creating-cluster
-  - Name: Accessing a ROSA cluster
-    File: rosa-accessing-cluster
-  - Name: Configuring identity providers using the Red Hat OpenShift Cluster Manager
-    File: rosa-config-identity-providers
+#  - Name: Accessing a ROSA cluster
```

Comment:
I commented out _Accessing a ROSA cluster_ and _Configuring identity providers using the Red Hat OpenShift Cluster Manager_ because they are duplicates of the assemblies in the _Installing, accessing, and deleting ROSA clusters_ directory. If everyone is okay with me removing those files, I will do so.